templates/design40_webpages/client_config/_bank.html

```diff
@@ -0,0 +1,28 @@
+[% USE HTML %]
+[% USE LxERP %]
+[% USE L %]
+[% USE T8 %]
+
+<div id="bank">
+<div class="wrapper">
+  <table class="tbl-horizontal">
+    <caption>[% 'SEPA' | $T8 %]</caption>
+    <tbody>
+      <tr>
+        <th>[% LxERP.t8('SEPA-XML File ending') %]</th>
+        <td>[% L.yes_no_tag('defaults.sepa_export_xml', SELF.defaults.sepa_export_xml) %]</td>
+        <td class="longdesc">[% LxERP.t8('If enabled the file ending for SEPA exports will be .xml instead of .cct or .cdd') %]</td>
+      </tr>
+    <caption>[% 'Bank' | $T8 %]</caption>
```

Comment:
Das funktioniert nicht. <caption> darf nur direkt hinter <table> kommen:
"If included, the <caption> element must be the first child of its parent <table> element." (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption)

Ich glaube HP hat für den fall, dass Du keine zweite Tabelle möchtest eine Klasse "caption" gemacht, die Du im <th> nehmen könntest. Must Du mal testen.